### PR TITLE
Correct some spelling errors in  `generate_dummy_segmentation_extractor`

### PR DESCRIPTION
**Pull request by h-mayorquin**
As in the title.

### CODE REVIEW

File: src/roiextractors/testing.py [hunk 1]
```diff
@@ -118,23 +118,23 @@ def generate_dummy_segmentation_extractor(
     num_rois : int, optional
         number of regions of interest, by default 10.
     num_frames : int, optional
-        _description_, by default 30
-    num_rows : number of frames used in the hypotethical video from which the data was extracted, optional
-        number of rows in the hypotethical video from which the data was extracted, by default 25.
+        description, by default 30.
```
Comment (pauladkisson) [inline src/roiextractors/testing.py:121]: Can you add a description instead of this placeholder?

**Review by pauladkisson — CHANGES_REQUESTED** [review]

File: src/roiextractors/testing.py (hunk 1 above)
Comment (h-mayorquin) [reply src/roiextractors/testing.py:121]: I just corrected the `_description_`to  `description`, what do you mean? was that doing something different?

[the remark this answers is not in the document]
Comment (pauladkisson) [reply src/roiextractors/testing.py:121]: > I just corrected the _description_to description, what do you mean? was that doing something different?

Like this

[the remark this answers is not in the document]
Comment (h-mayorquin) [reply src/roiextractors/testing.py:121]: My bad, obviously : P

Thanks!

**Review by pauladkisson — APPROVED** [review]

**Pull request merged**